frontend/js/src/explore/fresh-releases/FreshReleases.tsx

```diff
@@ -92,100 +170,133 @@ export default function FreshReleases() {
-                return (
-                  <ReleaseCard
-                    key={release.release_mbid}
-                    releaseDate={release.release_date}
-                    releaseMBID={release.release_mbid}
-                    releaseName={release.release_name}
-                    releaseTypePrimary={release.release_group_primary_type}
-                    releaseTypeSecondary={release.release_group_secondary_type}
-                    artistCreditName={release.artist_credit_name}
-                    artistMBIDs={release.artist_mbids}
-                    confidence={release.confidence}
-                    caaID={release.caa_id}
-                    caaReleaseMBID={release.caa_release_mbid}
+          ) : (
+            <div id="release-card-grids" ref={releaseCardGridRef}>
+              {filteredList.length === 0 ? (
+                <div id="no-release">
+                  <img
+                    src="/static/img/recommendations/no-freshness.png"
+                    alt="No Releases Found"
                   />
-                );
-              })}
-            </div>
+                </div>
+              ) : (
+                <ReleaseCardsGrid
+                  filteredList={filteredList}
+                  displaySettings={displaySettings}
+                  order={sort}
+                />
+              )}
 
-            {pageType === PAGE_TYPE_SITEWIDE ? (
-              <div className="releases-timeline col-xs-12 col-md-1">
-                {releases.length > 0 ? (
-                  <ReleaseTimeline releases={filteredList} />
-                ) : null}
-              </div>
-            ) : null}
-          </>
-        )}
+              {pageType === PAGE_TYPE_SITEWIDE && releases.length > 0 ? (
+                <div className="releases-timeline col-xs-12 col-md-1">
+                  <ReleaseTimeline releases={filteredList} order={sort} />
+                </div>
+              ) : null}
+            </div>
+          )}
+        </div>
+      </div>
+      <div
+        className={`release-filters ${
+          isSidebarOpen ? "open" : ""
+        } col-xs-12 col-md-2`}
+      >
+        <ReleaseFilters
+          allFilters={allFilters}
+          releases={releases}
+          setFilteredList={setFilteredList}
+          range={range}
+          handleRangeChange={handleRangeChange}
+          displaySettings={displaySettings}
+          toggleSettings={toggleSettings}
+          showPastReleases={showPastReleases}
+          setShowPastReleases={setShowPastReleases}
+          showFutureReleases={showFutureReleases}
+          setShowFutureReleases={setShowFutureReleases}
+          releaseCardGridRef={releaseCardGridRef}
+          pageType={pageType}
+        />
       </div>
-    </>
+      <button
+        className="toggle-sidebar-button"
+        style={{
+          backgroundColor: isSidebarOpen ? "#353070" : "#fff",
+          color: isSidebarOpen ? "#fff" : "#353070",
+        }}
```

Comment:
These would be better as CSS classes if we can.
Perhaps an `.open` class like you did for the sidebar itself? or `.active` ?